### PR TITLE
added Nextflow Tower and Galaxy as integration partners

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
A description of the PR, should include a decent explanation as to why this change was needed and a decent explanation as to what this change does
Screenshot from 2022-09-14 15-50-58

**Issue**
A link to a github issue or SEAB- ticket (using that as a prefix)
https://ucsc-cgl.atlassian.net/browse/DOCK-1806
https://github.com/dockstore/dockstore/issues/4266

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

### CODE REVIEW

**Review by denis-yuen — COMMENTED** [review]
Can we get a screenshot, it could probably just fill in the description for you

**Comment by hyunnaye** [comment]
@denis-yuen screenshot added into the PR description

**Review by denis-yuen — APPROVED** [review]

File: src/app/about/about.component.ts [hunk 1]
```diff
@@ -27,6 +27,8 @@ export class AboutComponent implements OnInit {
     new Sponsor('terra.png', new URL('https://terra.bio')),
```
Comment (coverbeck) [inline src/app/about/about.component.ts:27]: Outside the scope of this PR, but happened to notice on line 25 -- @denis-yuen , now that we removed the launch with DNAstack button, should we remove them as an integration partner as well?

Comment (denis-yuen) [reply src/app/about/about.component.ts:27]: I'd say leave it. They're not a "launch-with" partner in the sense of having a button but they're an integration partner in the sense that workbench uses our API and advertises us.

**Review by coverbeck — APPROVED** [review]

**Pull request merged**